[EDU-4528] feat: add live streaming use case - architecture (#934)
aziontech/docs: src/content/docs/pt-br/pages/arquiteturas/live-streming-delivery.mdx

---
title: Otimize a distribuição de conteúdo de vídeo com entrega de streaming ao vivo
description: Esta arquitetura aproveita recursos avançados como transcodificação em tempo real, streaming adaptativo de taxa de bits e armazenamento em cache inteligente, possibilitando experiências impecáveis para os espectadores.
meta_tags: architecture, content delivery, hls, edge computing
namespace: docs_arch_live_streaming_delivery
permalink: /documentacao/arquitetura/live-streaming-delivery/live-streaming-delivery-com-hls/
---

> A edge computing transforma a [entrega de streaming ao vivo](ADD LINK***) (*live streaming delivery*) otimizando a distribuição de conteúdo de vídeo de alta qualidade. Esta arquitetura aproveita recursos avançados como transcodificação em tempo real e streaming adaptativo de taxa de bits, que, juntamente com o armazenamento em cache inteligente, possibilitam experiências impecáveis para os espectadores. A natureza distribuída da edge computing permite que o conteúdo seja processado e armazenado mais próximo do usuário, reduzindo a latência e as restrições de largura de banda, garantindo uma transmissão de alta qualidade mesmo durante períodos de alta demanda.

Author: ```suggestion
A edge computing transforma a [entrega de streaming ao vivo](https://www.azion.com/pt-br/solucoes/live-streaming/) (*live streaming delivery*) otimizando a distribuição de conteúdo de vídeo de alta qualidade. Esta arquitetura aproveita recursos avançados como transcodificação em tempo real e streaming adaptativo de taxa de bits, que, juntamente com o armazenamento em cache inteligente, possibilitam experiências impecáveis para os espectadores. A natureza distribuída da edge computing permite que o conteúdo seja processado e armazenado mais próximo do usuário, reduzindo a latência e as restrições de largura de banda, garantindo uma transmissão de alta qualidade mesmo durante períodos de alta demanda.
```

    - Em seguida, crie uma política de cache para a playlist.
        - Repita o processo anterior, mas desta vez defina um **Maximum TTL** de `0` para **Browser Cache Settings** e `5` para **Edge Cache Settings**. Defina também as configurações da **Advanced Cache Key**.
    - Ainda na página da aplicação, abra a aba **Rules Engine**:
        - Crie uma regra para os chunks: selecione `if ${uri}` *matches* `.\*.ts` como critério (criteria) e **Set Cache Policy** como comportamento (behavior), adicionando a política para os chunks criada no passo anterior.
        - Crie uma regra para a playlist: defina `if ${uri}` *matches* `\*.m3u8` como critério (criteria) e **Set Cache Policy** como comportamento (behavior), adicionando a política para a playlist criada no passo anterior.
4. Configure sua fonte e codificador apontando para as entradas DNS de ingestão da Azion.
5. Transmita seu conteúdo via Plataforma de Edge da Azion.

---

## Documentação relacionada

- [Azion Edge Network](https://www.azion.com/pt-br/produtos/edge-network/)
- [Cache Settings](/pt-br/documentacao/produtos/build/edge-application/cache-settings/)

> - [Content delivery - Architecture](ADD LINK***)

Author: ```suggestion
- [Content delivery - Architecture](/pt-br/documentacao/arquitetura/edge-application/content-delivery/)
```

> - [Content delivery - Architecture](/pt-br/documentacao/arquitetura/edge-application/content-delivery/)

Author: ```suggestion
- [Content delivery architecture](/pt-br/documentacao/arquitetura/edge-application/content-delivery/)
```
